packages/components/src/components/mobile-dialog/mobile-dialog.scss

```diff
@@ -93,7 +93,21 @@
                         border-bottom: 0.01rem solid transparent;
 
                         .inline-message__information {
-                            margin: 1.6rem 0.8rem;
+                            margin: 1.6rem 0.8rem -0.8rem;
+                        }
+
+                        .contract-type__learn-more {
```

Comment:
Component namings should be generalized. We should not couple components with business names. So, can we remove the `contract-type` from here?